paasta_tools/utils.py

```diff
@@ -257,13 +257,13 @@ def format_docker_parameters(self, with_labels=True):
         parameters.extend(self.get_cap_add())
         return parameters
 
-    def get_disk(self):
+    def get_disk(self, default=1024):
```

Comment:
I don't know why you'd want to parameterize this? if you want a specific value, set it in your config?